.github/workflows/nightly.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       uses: actions/checkout@v4
       with:
         repository: sbt/zinc
-        ref: develop
+        ref: 1.10.x
```

Comment:
Probably we should revert this

Reply:
Addressed in 12db63951b9a34c196622dbf7b8a7fdfe679435a